examples/tutorials/contour-map.py

```diff
@@ -0,0 +1,73 @@
+"""
+Creating a map with contour lines
+=================================
+
+Plotting a contour map is handled by :meth:`pygmt.Figure.grdcontour`
+"""
+
+import pygmt
+
+########################################################################################
+# Create contour plot
+# -------------------
+#
+# The grdcontour method takes the grid input and the region values.
```

Comment:
This sentence gives me the feeling that `region` is a required argument, but it's not.
```suggestion
# The :meth:`pygmt.Figure.grdcontour` method takes the grid input.
```